generator/.DevConfigs/petesong-sso-token-fix.json

```diff
@@ -0,0 +1,9 @@
+{
+    "services": [
+      {
+        "serviceName": "SSO OIDC",
```

Comment:
I ran the dry-run immediately after pushing a new revision and it failed on the merge step. I think it might've been because of the service name having the space but I'll need to see.